samples/containers/android/src/main/java/com/squareup/sample/container/panel/PanelOverlayDialogFactory.kt

```diff
@@ -0,0 +1,52 @@
+package com.squareup.sample.container.panel
+
+import android.app.Dialog
+import android.graphics.drawable.ColorDrawable
+import android.util.TypedValue
+import android.view.View
+import com.squareup.sample.container.R
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.container.Bounds
+import com.squareup.workflow1.ui.container.ModalScreenOverlayDialogFactory
+import com.squareup.workflow1.ui.container.setBounds
+
+@OptIn(WorkflowUiExperimentalApi::class)
+internal object PanelOverlayDialogFactory : ModalScreenOverlayDialogFactory<PanelOverlay<*>>(
```

Comment:
Holding off on kdoc for the Panel stuff until I polish it up in the next PR.

Reply:
Gonna push back. This is sample code, not for general use. I'm even tempted to rename it just `PanelDialogFactory`. Regardless, inclined to hold off on painting this part of the bikeshed until the next PR, with improved Panel fu.

Reply:
Ya - I can't settle on whether I think of this as _just_ sample code or as paradigmatic samples that act like documentation. But your point stands and is likely the most important one. Keep it simple and don't let perfect be the enemy of good.